neow3j-test/src/test/java/io/neow3j/test/ExampleTest.java

```diff
@@ -0,0 +1,53 @@
+package io.neow3j.test;
+
+import io.neow3j.contract.SmartContract;
+import io.neow3j.protocol.Neow3jExpress;
+import io.neow3j.protocol.core.response.NeoApplicationLog;
+import io.neow3j.transaction.AccountSigner;
+import io.neow3j.types.Hash256;
+import io.neow3j.utils.Await;
+import io.neow3j.wallet.Account;
+import org.junit.Ignore;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import java.math.BigInteger;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@Ignore
```

Comment:
ahh... I get it... this annotation is used just to tell JUnit to exclude the tests from this class (if executed from gradle). Or is it used for something else?

Reply:
I only added this because this test was still in unit tests even though it's an integration test. I moved it to an integrationTest folder now and removed the `@Ignore`